### PR TITLE
challenge-one-task

### PR DESCRIPTION
**Pull request by sunk0**
challenge-one-task

### CODE REVIEW

File: public/challenge-contributions/1/asavovStyles.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+html {
```
Comment (rumenpetrov) [inline public/challenge-contributions/1/asavovStyles.css:1]: The CSS should be in the HTML file.

image

File: public/challenge-contributions/1/asavov.html [hunk 2]
```diff
@@ -4,8 +4,46 @@
   <meta charset="UTF-8">
```
Comment (rumenpetrov) [inline public/challenge-contributions/1/asavov.html:4]: Please validate your HTML, there is a minor thing you should update. https://validator.w3.org/#validate_by_input

**Review by rumenpetrov — CHANGES_REQUESTED** [review]
I added 2 small change request. Please take a look and you will be good to go. ;)

File: public/challenge-contributions/1/asavov.html (hunk 2 above)
Comment (sunk0) [reply public/challenge-contributions/1/asavov.html:4]: sorry but I don't understand what you actually want here

**Review by rumenpetrov — APPROVED** [review]
Looks good, thank you the updates.

**Review by rumenpetrov — APPROVED** [review]
You made it! :) You got everything covered. Thanks for the update and sorry for the misunderstanding.

**Pull request merged**